maven-plugin/src/main/java/com/marvinformatics/formatter/model/RuleSet.java

```diff
@@ -36,16 +36,16 @@
 	 */
 	@Override
 	public void addRuleInstances(Digester digester) {
-		digester.addObjectCreate("profiles", Profiles.class);
-		digester.addObjectCreate("profiles/profile", Profile.class);
-		digester.addObjectCreate("profiles/profile/setting", Setting.class);
+	 	digester.addObjectCreate("profiles", Profiles.class);
+	 	digester.addObjectCreate("profiles/profile", Profile.class);
+	 	digester.addObjectCreate("profiles/profile/setting", Setting.class);
```

Comment:
Define a constant instead of duplicating this literal "profiles/profile/setting" 4 times.